Documentation/Gifbuilder/Gifbuilder/Index.rst

```diff
@@ -1,73 +1,75 @@
+and they will be rendered onto an image one by one.
 
-The name "GIFBUILDER" comes from the time where GIF was the only file
-format supported. PNG and JPG can just as well be created today
-(configured with $TYPO3\_CONF\_VARS['GFX']).
+The name "GIFBUILDER" comes from the time when GIF was the only supported file
+format. PNG and JPG can be created as well today (configured with
+:ref:`$TYPO3_CONF_VARS['GFX'] <t3coreapi:typo3ConfVars_gfx>`).
 
 
 .. _gifbuilder-calc:
 
 Note on (+calc)
 ===============
 
-Whenever the "+calc"-function is added to a value in the data type of
-the properties underneath, you can use the dimensions of TEXT and
-IMAGE objects from the Gifbuilder Object array. This is done by
-inserting a tag like this: "[10.w]" or "[10.h]", where "10" is the
-Gifbuilder Object number in the array and "w"/"h" signifies either
-width or height of the object.
-
-The special property "lineHeight" (e.g. "[10.lineHeight]") uses the
-height a single line of text would take.
-
-On using the special function max(), the maximum of multiple values
-can be determined. Example:
-
-.. code-block:: typoscript
-   :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
-
-   XY = [10.w]+[20.w], max([10.h], [20.h])
-
-Here's a full example (taken from "styles.content (default)"):
-
-
-.. code-block:: typoscript
-   :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
-   :emphasize-lines: 6
-
-   styles.header.gfx1 = IMAGE
-   styles.header.gfx1 {
-     wrap = {$styles.header.gfx1.wrap}
-     file = GIFBUILDER
-     file {
-       XY = [10.w]+10 ,{$styles.header.gfx1.itemH}
-       backColor = {$styles.header.gfx1.bgCol}
-       10 = TEXT
-       10 {
-         text.current = 1
-         text.crop = {$styles.header.gfx1.maxChars}
-         fontSize = {$styles.header.gfx1.fontSize}
-         fontFile = {$styles.header.gfx1.file.fontFile}
-         fontColor = {$styles.header.gfx1.fontColor}
-         offset = {$styles.header.gfx1.fontOffset}
-       }
-     }
-   }
+Whenever the :ref:`+calc <calc>` function is added to a value in the data type
+of the properties underneath, you can use the dimensions of
+:ref:`TEXT <gifbuilder-text>` and :ref:`IMAGE <gifbuilder-image>` objects from
+the GIFBUILDER object array. This is done by inserting a tag like this:
+:typoscript:`[10.w]` or :typoscript:`[10.h]`, where `10` is the GIFBUILDER
+object number in the array and `w`/`h` signifies either width or height of the
+object.
+
+The special property :typoscript:`lineHeight` (for example,
+:typoscript:`[10.lineHeight]`) uses the height a single line of text would take.
+
+On using the special function :typoscript:`max()`, the maximum of multiple
+values can be determined. Example:
+
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/TypoScript/setup.typoscript
+
+    XY = [10.w]+[20.w], max([10.h], [20.h])
+
+Here's a full example (taken from "styles.content (default)" of older TYPO3
```

Comment:
Not sure how helpful this example is?

Reply:
At least it is an example. I never worked with GIFBUILDER in the past, so have not really a clue what is going on here. The whole working from me on GUIFBUILDER now is preparing it for adding a changelog entry for v13 (support of webp).

The example can be tackled in a separate PR.

Reply:
ack